.github/workflows/ci.yml

```diff
@@ -16,18 +29,59 @@ jobs:
           node-version-file: 'package.json'
       - run: node --version
       - run: npm ci
-      # building the static site (Next.js) will fail if linting fails, so explicitly run linting
-      # first to make the test output clearer
```

Comment:
From c305e916d03d794f35d6483e0c7b40703ba560aa:

> it doesn't make sense anymore to lint _after_ build

I assume you meant _before_ build?

Reply:
Ah, this was poor wording on my part. I'll fix it up with this:

```diff
-    Lint is moved out of test, because it doesn't make sense anymore to lint
-    _after_ build (since, as a previous comment noted, the build will fail
-    if linting fails).
+    Lint is moved out of test, because test now happens after build and it
+    doesn't make sense to lint _after_ build (since, as a previous comment
+    noted, the build will fail if linting fails).
```

Reply:
Rewording in [re-push](https://tsibley.net/paste/2024-05-13mFnDJvko-diff.html).